ofl/alumnisans/METADATA.pb

```diff
@@ -32,221 +32,3 @@ axes {
-languages: "nso_Latn"  # Northern Sotho
-languages: "ny_Latn"  # Nyanja
-languages: "nyn_Latn"  # Nyankole
-languages: "oc_Latn"  # Occitan
-languages: "om_Latn"  # Oromo
-languages: "pam_Latn"  # Pampanga
-languages: "pap_Latn"  # Papiamento
-languages: "pau_Latn"  # Palauan
-languages: "pcd_Latn"  # Picard
-languages: "pko_Latn"  # Pökoot
-languages: "pl_Latn"  # Polish
-languages: "pms_Latn"  # Piedmontese
-languages: "pon_Latn"  # Pohnpeian
-languages: "pt_Latn"  # Portuguese
-languages: "quc_Latn"  # Kʼicheʼ
-languages: "qug_Latn"  # Chimborazo Highland Quichua
-languages: "rar_Latn"  # Rarotongan
-languages: "rm_Latn"  # Romansh
-languages: "rn_Latn"  # Rundi
-languages: "ro_Latn"  # Romanian
-languages: "rof_Latn"  # Rombo
-languages: "ru_Cyrl"  # Russian
-languages: "rup_Latn"  # Aromanian
-languages: "rw_Latn"  # Kinyarwanda
-languages: "rwk_Latn"  # Rwa
-languages: "saq_Latn"  # Samburu
-languages: "sbp_Latn"  # Sangu
-languages: "sc_Latn"  # Sardinian
-languages: "scn_Latn"  # Sicilian
-languages: "sdc_Latn"  # Sassarese Sardinian
-languages: "se_Latn"  # Northern Sami
-languages: "seh_Latn"  # Sena
-languages: "sei_Latn"  # Seri
-languages: "sg_Latn"  # Sango
-languages: "sk_Latn"  # Slovak
-languages: "sl_Latn"  # Slovenian
-languages: "sm_Latn"  # Samoan
-languages: "sma_Latn"  # Southern Sami
-languages: "smj_Latn"  # Lule Sami
-languages: "smn_Latn"  # Inari Sami
-languages: "sn_Latn"  # Shona
-languages: "snk_Latn"  # Soninke
-languages: "so_Latn"  # Somali
-languages: "sq_Latn"  # Albanian
-languages: "sr_Cyrl"  # Serbian
-languages: "sr_Latn"  # Serbian (Latin)
-languages: "srn_Latn"  # Sranan Tongo
-languages: "ss_Latn"  # Swati
-languages: "su_Latn"  # Sundanese
-languages: "sv_Latn"  # Swedish
-languages: "sw_Latn"  # Swahili
-languages: "swb_Latn"  # Comorian, Latin
-languages: "szl_Latn"  # Silesian
-languages: "teo_Latn"  # Teso
-languages: "tet_Latn"  # Tetum
-languages: "tiv_Latn"  # Tiv
-languages: "tk_Arab"  # Turkmen
-languages: "tk_Latn"  # Turkmen, Latin
-languages: "tkl_Latn"  # Tokelau
-languages: "tn_Latn"  # Tswana
-languages: "tpi_Latn"  # Tok Pisin
-languages: "tr_Latn"  # Turkish
-languages: "ts_Latn"  # Tsonga
-languages: "tum_Latn"  # Tumbuka
-languages: "tvl_Latn"  # Tuvalu
-languages: "ty_Latn"  # Tahitian
-languages: "ug_Latn"  # Uyghur, Latin
-languages: "umb_Latn"  # Umbundu
-languages: "vec_Latn"  # Venetian
-languages: "vep_Latn"  # Veps
-languages: "vi_Latn"  # Vietnamese
-languages: "vmw_Latn"  # Makhuwa
-languages: "vo_Latn"  # Volapük
-languages: "vro_Latn"  # Võro
-languages: "vun_Latn"  # Vunjo
-languages: "wa_Latn"  # Walloon
-languages: "wae_Latn"  # Walser
-languages: "war_Latn"  # Waray
-languages: "wbp_Latn"  # Warlpiri
-languages: "wo_Latn"  # Wolof
-languages: "xav_Latn"  # Xavánte
-languages: "xh_Latn"  # Xhosa
-languages: "xog_Latn"  # Soga
-languages: "yao_Latn"  # Yao
-languages: "yap_Latn"  # Yapese
-languages: "yua_Latn"  # Yucateco
-languages: "zu_Latn"  # Zulu
```

Comment:
@m4rc1e I saw the same issue in another PR, what do we do until fixed? We re-add the languages manually?

Reply:
Yes, please readd them. I have an issue open https://github.com/googlefonts/gftools/issues/456. Personally, I wouldn't PR any family upgrade until this is fixed.